services/i18n/src/init.js

```diff
@@ -0,0 +1,26 @@
+import i18next from 'i18next'
+import moment from 'moment'
+import { defaultNamespace } from './defaultNamespace'
+
+export const init = () => {
```

Comment:
Would be nice to expose the set up of the init properties to the user. As an example, that would allow the customization of the namespace separator without needing to fork the service.

Reply:
@sferadev Thanks for the input!  This is still a work in progress, so we might support customized initialization.  However we don't want to leak the i18next internal implementation detail, so we're locking down parts of the API.  **Can you share the use case you envision for wanting to pass a custom namespace separator?**   With the approach as-is namespaces are usually specified through React context, so they wouldn't need to be specified with each translate call.

Reply:
Haven't tried this particular implementation yet but I recall time ago we had problems with the namespace separator on d2-i18n and had to either avoid certain characters or change the namespace separator as parts of the strings were masked out from the pot files.

Reply:
Got it, that makes sense.  I've been looking into switching from sentence-keyed translations to hash-keyed translations which would avoid this problem altogether, but that might still be a ways off.  We're going to hold off on overhauling the i18n system for now.